js/index.js

```diff
@@ -38,5 +67,15 @@ window.addEventListener('load', () => {
   const calculatePricesBtn = document.getElementById('calculate');
   calculatePricesBtn.addEventListener('click', calculateAll);
 
-  //... your code goes here
+  const removeButtons = document.getElementsByClassName('btn btn-remove');
```

Comment:
eu aaaacho que se remover o btn fica melhor. Então ficaria document.getElementsByClassName('btn-remove') ou querySelectorAll('.btn-remove').

Não sei se esse é o problema, mas se testar e não for podem vir falar comigo